reporter/resource/reuters.py

```diff
@@ -49,52 +49,63 @@ def download_prices_from_reuters(username: str,
                                  password: str,
                                  dest_dir: Path,
                                  rics: List[str],
-                                 logger: Logger) -> None:
+                                 logger: Logger = None,
+                                 start: str = None,
+                                 end: str = None) -> None:
 
     logging.getLogger('urllib3').setLevel(logging.WARNING)
     logging.getLogger('requests').setLevel(logging.CRITICAL)
 
     dest_dir.mkdir(parents=True, exist_ok=True)
-
-    for ric in rics:
-
-        filename = ric2filename(dest_dir, ric, 'csv.gz')
-
-        if filename.is_file():
-            continue
-
-        logging.info('start downloading {} from Reuters'.format(ric))
-
-        auth_token = get_auth_token(username, password)
-        payload = make_payload(ric)
-        header = make_extract_header(auth_token)
-
-        r = requests.post(Reuters.extraction_raw_uri,
-                          data=None,
-                          json=payload,
-                          headers=header)
-
-        if r.status_code != HTTPStatus.OK.value:
-
-            if r.status_code != HTTPStatus.ACCEPTED.value:
-                return
-
-            location = r.headers['Location']
-            while True:
-                r = requests.get(location, headers=header)
-                if r.status_code == HTTPStatus.OK.value:
-                    break
-                else:
-                    time.sleep(5)
-
-        job_id = r.json().get('JobId')
-        results_uri = Reuters.raw_extraction_results_uri.format(job_id)
-        r = requests.get(results_uri, headers=header, stream=True)
-
-        with filename.open(mode='wb') as f:
-                f.write(r.raw.read())
-
-        logging.info('end downloading {} from Reuters'.format(ric))
+    reqs = []
+    auth_token = get_auth_token(username, password)
+    header = make_extract_header(auth_token)
+
+    todo = [{
+            'ric': ric,
+            'uri': Reuters.extraction_raw_uri,
+            'file': ric2filename(dest_dir, ric, 'csv.gz')
+        } for ric in rics]
```

Comment:
Please fix a PEP8 error, `E124 closing bracket does not match visual indentation`.
```suggestion
            } for ric in rics]
```